.github/workflows/main.yml

```diff
@@ -113,9 +113,12 @@ jobs:
     name: Go test e2e
     runs-on: ubuntu-latest
     needs: build
+    strategy:
+      matrix:
+        bucket: [1, 2]
     env:
       CTR_TAG: ${{ github.sha }}
-      CTR_REGISTRY: "localhost:5000"   # unused for kind, but currently required in framework
+      CTR_REGISTRY: "localhost:5000" # unused for kind, but currently required in framework
```

Comment:
We could probably set a meaningful default wherever this is used and remove it from here...

Reply:
Yeah we could easily add a check to the framework to add some default if kind is being used. I'll plan to add that.